packages/session-replay-browser/src/hooks/click.ts

```diff
@@ -102,14 +105,16 @@ export const clickHook: (logger: ILogger, options: Options) => mouseInteractionC
 
     const { left: scrollX, top: scrollY } = utils.getWindowScroll(globalScope as unknown as Window);
 
+    const maskedPageUrl = getPageUrl(logger, location.href, ugcFilterRules);
```

Comment:
Nit: Since this may or may not be masked.
```suggestion
    const pageUrl = getPageUrl(logger, location.href, ugcFilterRules);
```